### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by WouterJD** [WouterJD/FortiusANT#33]
@fritz-hh FortiusANT is depending on usb (import usb.core) and fxload.
Also, on python version 3

Is this something that can be defined in requirements.txt?

Thanks in advance for info!

**Comment by fritz-hh** [comment]
The short answer, is: I have no experience with python...

The longer answer:
* I did not find any module called `usb` in pip, and I had to remove it from the requirement file (installing the depencies lead to a crashes before I removed `usb`).
From what I understood, `usb.core` is provided by the `pyusb` module, that you already have in the requirements file.
* I do not think that you have an external dependency to fxload, as `fxload.py` is part of your code: https://github.com/WouterJD/FortiusANT/blob/master/pythoncode/fxload.py
* For python 3, I do not know.
